.gitattributes

```diff
@@ -1,2 +1,13 @@
+* text=auto
```

Comment:
Wildcard `text=auto` is rather dangerous, IMO.  It will happily mangle binary files if git perceives them to be text files and can be brutal to debug.  I'd use a whitelisting approach, like:
```
*.js text=auto
*.md text=auto  # just an example, don't know if this should apply to *.md 
```